dbusmock/templates/bluez5.py

```diff
@@ -253,6 +259,47 @@ def AddAdapter(self, device_name, system_name):
         ],
     )
 
+    bluez = mockobject.objects["/org/bluez"]
+
+    # Advertising Manager
+    if bluez.enable_advertise_api:
+        max_advertisement_instances = 5
+        advertising_manager_properties = {
+            "ActiveInstances": dbus.Byte(0, variant_level=1),
+            "SupportedInstances": dbus.Byte(max_advertisement_instances, variant_level=1),
+            "SupportedIncludes": dbus.Array(["tx-power", "appearance", "local-name", "rssi"], variant_level=1),
+            "SupportedSecondaryChannels": dbus.Array(["1M", "2M", "Coded"], variant_level=1),
+            "SupportedCapabilities": dbus.Dictionary(
+                {
+                    "MaxAdvLen": dbus.Byte(251),
+                    "MaxScnRspLen": dbus.Byte(251),
+                    "MinTxPower": dbus.Int16(-34),
+                    "MaxTxPower": dbus.Int16(7),
```

Comment:
Do these numbers have any meaning and come from somewhere? (Like a particular device) or just random?

Reply:
The TX power values are in dbm and I believe are regulated depending on BT device type. I've grabbed these and the other values them from my bluetooth adapter (Intel AX200). Happy to document that if needed.

Reply:
Added a brief comment about the source of the values.